network/core-node/admin-guide/publishing-history-archives.mdx

```diff
@@ -3,47 +3,39 @@ title: Publishing History Archives
 sidebar_position: 50
 ---
 
-import { CodeExample } from "@site/src/components/CodeExample";
-
-If you want to run a [Full Validator](../README.mdx#full-validator) or an [Archiver](../README.mdx#archiver), you need to set up your node to publish a history archive. You can host an archive using a blob store such as Amazon's S3 or Digital Ocean's spaces, or you can simply serve a local archive directly via an HTTP server such as Nginx or Apache. If you're setting up a [Basic Validator](../README.mdx#basic-validator), you can skip this section. No matter what kind of node you're planning to run, make sure to set it up to `get` history, which is covered in [Configuration](./configuring.mdx).
+If you want to run a [Full Validator](../README.mdx#full-validator) or an [Archiver](../README.mdx#archiver), you need to set up your node to publish a history archive. You can host an archive using a blob store such as Amazon's S3 or Digital Ocean's spaces, or you can simply serve a local archive directly via an HTTP server such as Nginx or Apache. If you're setting up a [Basic Validator](../README.mdx#basic-validator), you can skip this section. No matter what kind of node you're planning to run, make sure to set it up to `get` history, which is covered in [Environment Preparation](./environment-preparation.mdx).
 
 ## Caching and History Archives
 
-You can significantly reduce the data transfer costs associated with running a public History archive by using common caching techniques or a CDN.
-
-Three simple rules apply to caching the History archives:
+You can significantly reduce the data transfer costs associated with running a public History archive by using common caching techniques or a CDN. Three simple rules apply to caching the History archives:
```

Comment:
Do we want to point out that bandwidth costs for egress are usually the most expensive part of running a validator?  And that caching is crucial to keep those costs down?  I think something along those lines might be helpful